### DISCUSSION TITLE
introduction/quick-start

### DISCUSSION THREAD

**Discussion by giscus** [linpeilie/mapstruct-plus#15] [Announcements]
# introduction/quick-start

MapStructPlus 快速开始教程 quick start

https://mapstruct.plus/introduction/quick-start.html

**Comment by noday** [comment]
如果pom里的annotationProcessorPaths可以去掉，convert改为静态类方法那就更完美了

**Reply by noday** [reply]
怎么样，整一下

**Reply by linpeilie** [reply]
1. annotationProcessorPaths 倒是可以做一下，但现在项目用 Lombok 的比较多，如果要整合一起用， 那就必须有 annotationProcessorPaths 配置；
2. 可以增加一个静态工具类，注入 Converter 实例，来提供静态方法

**Reply by zhyblack** [reply]
idea lombok 和 mybatis-flex 自动生成 是可以不用配置 annotationProcessorPaths，引入 MapstructPlus，需要配置annotationProcessorPaths 而且还要把 lombok 和 mybatis-flex 一起配置，希望可以优化一下，

**Reply by linpeilie** [reply]
lombok和mybatis-flex之间并不会互相影响，或者说没有先后顺序要求，所谓的解决方式，就是在依赖中，添加processor包，这并不是解决了，Lombok 和 MapStruct 之间是要有先后顺序要求的，所以必须要在 annotationProcessorPaths 中指定注解处理器的执行顺序。

**Reply by zhyblack** [reply]
明白了，好的👌
